bazelisk.go

```diff
@@ -566,6 +579,45 @@ func downloadBazel(fork string, version string, isCommit bool, directory string)
 	return destinationPath, nil
 }
 
+func copyFile(src, dst string, perm os.FileMode) (err error) {
```

Comment:
Why is this a named return parameter? This feature is not being used in the function body (i.e. there aren't any bare return statements).

Reply:
You are right. The named return parameter is unnecessary.